Dockerfile

```diff
@@ -1,42 +1,10 @@
-# Simple usage with a mounted data directory:
-# > docker build -t simapp .
-#
-# Server:
-# > docker run -it -p 26657:26657 -p 26656:26656 -v ~/.simapp:/root/.simapp simapp simd init test-chain
-# TODO: need to set validator in genesis so start runs
-# > docker run -it -p 26657:26657 -p 26656:26656 -v ~/.simapp:/root/.simapp simapp simd start
-#
-# Client: (Note the simapp binary always looks at ~/.simapp we can bind to different local storage)
-# > docker run -it -p 26657:26657 -p 26656:26656 -v ~/.simappcli:/root/.simapp simapp simd keys add foo
-# > docker run -it -p 26657:26657 -p 26656:26656 -v ~/.simappcli:/root/.simapp simapp simd keys list
-# TODO: demo connecting rest-server (or is this in server now?)
-FROM golang:1.18-alpine AS build-env
+# This docker image is designed to be used in CI for benchmarks and also by developers wanting an environment that always has the lastest rocksdb and cleveldb. 
+# Since the SDK is not simapp, this Dockerfile doesn't build simapp, but instead provides a full development environment for the Cosmos SDK.
+# Other Docker images, like simapp, or the various protobuf build iamges could be based on this one
+# This Docker image is rolling release, like most developer laptops.  Things will break if upstream changes and we've not kept up do date.  That's intentional.
+# This docker image is made here, every 24 hours: https://github.com/faddat/archlinux-docker
+# Compared to Alpine, this image may be larger, but for example, we do not need to shim glibc onto musl
 
-# Install minimum necessary dependencies
-ENV PACKAGES curl make git libc-dev bash gcc linux-headers eudev-dev python3
-RUN apk add --no-cache $PACKAGES
+FROM faddat/archlinux
```

Comment:
please use default image.

Reply:
it doesn't exist, ref #11688 

But what I can do is move the build process for the one above (faddat/archlinux) into this repo

Reply:
I'd like to explain this a little more.  I belive alpine is a very bad choice for containers because of musl by default.

In some of our tooling we install glibc into an alpine container, and then sometimes we:

https://github.com/CosmWasm/wasmd/pull/790/commits/35d86bd0f3b5db32d03217d919cde49fbdcd87ae

other times we forget and cause consensus issues in prod

so my thinking is to use a glibc distro

* ubuntu and debian ship very far out of date go, etc
* alpine: musl and need to build rocks and leveldb so in the terndermint repository for example the dockerfile switches over to debian for the e2e tests in order to use precompiled binaries of rocks and leveldb, which are out of date
* arch: unfortunately arch is two separate things:
archlinux.org
archlinuxarm.org

faddat/archlinux is a fork of a fork of a fork of tooling that takes the filesystem root of both amd64 and arm64 and mooshes them together

could just call it cosmos/linux

currently I have it set to build every hour and the dependencies are always up to date because of rolling releases.

end result: benchmarks that tell us the truth and much easier management